src/html/htmlcell.cpp

```diff
@@ -390,21 +401,30 @@ void wxHtmlWordCell::Split(const wxDC& dc,
     pos2 = j;
 
     wxASSERT( pos2 >= pos1 );
+
+    ext1 = pos1 == 0 ? 0 : (pos1 < widths.size() ? widths[pos1-1] : widths.Last());
```

Comment:
Is it actually possible to have `pos1 > widths.size()` here? It would be nice to have a comment explaining why is it OK for this to happen and why should we use the last elements of `widths` in this case (and also why we can be sure that `widths` is not empty).

And if not, i.e. if we always have `pos1 <= widths.size()`, we don't need to check for the condition here.